Chromium-XP_49_0_2623_113_VS2013_SSE3.txt

```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = Chromium-XP
+Version = 49.0.2623.113 VS2013 SSE3 1st Release
```

Comment:
```suggestion
Version = 49.0.2623.113 (VS2013 SSE3, 1st Release)
```

Reply:
I actually do not want to make the version string even longer, but I would agree to strip "1st Release" and "2nd Release" entirely, if that would be "good too".

Reply:
It would be fine, yes.

Reply:
Done, resolved.